packages/builder/src/pages/builder/app/[application]/design/[screenId]/_components/ComponentList/ComponentTree.svelte

```diff
@@ -89,6 +89,15 @@
     }
     return findComponentPath($selectedComponent, component._id)?.length > 0
   }
+
+  const handleMouseover = componentId => {
+    $store.hoverComponentId = componentId
```

Comment:
We should check here if the store already has this component hovered, and skip if so. Same applies to the other handlers in other files.